extensions-marketplace/develop/integrate/pinpoint.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: "PinPoint"
```

Comment:
For consistency with the [Extensions Gallery](https://www.agora.io/en/agora-extensions-marketplace/), please change it to "Bose PinPoint". This is what's agreed by our partner

Reply:
Done.